.manifest/sandbox/selectors/selectors.json

```diff
@@ -0,0 +1,241 @@
+{
+    "mainnets": {
```

Comment:
I think it would be better to use the raw yaml so we don't have to sync everything while people migrate, but this was the easiest PoC